plugins/dell/fu-plugin-dell.c

```diff
@@ -1114,11 +1114,23 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
-gboolean
-fu_plugin_coldplug (FuPlugin *plugin, GError **error)
+static gboolean
+fu_dell_coldplug (FuPlugin *plugin, GError **error)
```

Comment:
I think this should be `fu_plugin_dell_coldplug `per the naming scheme adopted for the others.